javacord-core/src/main/java/org/javacord/core/entity/activity/ActivityImpl.java

```diff
@@ -47,7 +56,10 @@ public ActivityImpl(DiscordApiImpl api, JsonNode data) {
         this.state = data.has("state") ? data.get("state").asText(null) : null;
         this.party = data.has("party") ? new ActivityPartyImpl(data.get("party")) : null;
         this.assets = data.has("assets") ? new ActivityAssetsImpl(this, data.get("assets")) : null;
+        this.secrets = data.has("secrets") ? new ActivitySecretsImpl(this, data.get("secrets")) : null;
         this.applicationId = data.has("application_id") ? data.get("application_id").asLong() : null;
+        this.createdAt = OffsetDateTime.parse(data.get("created_at").asText()).toInstant();
```

Comment:
I don't think that will work
```suggestion
        this.createdAt = Instant.ofEpochMilli(data.get("created_at").asLong());
```